tests/test_colocation_auto.py

```diff
@@ -195,7 +196,12 @@ def test_Colocator_update(what,raises):
 def test_Colocator_run_gridded_gridded(tm5_aero_stp):
     col = Colocator(**tm5_aero_stp)
     col.obs_id = col.model_id
-    col.run()
+    with catch_warnings():
```

Comment:
Wondering if some of these iris warnings will be gone once we adapt to the new version (i.e. after fixing #440)